openlibrary/plugins/upstream/account.py

```diff
@@ -1134,14 +1134,26 @@ class my_follows(delegate.page):
     path = r"/people/([^/]+)/(followers|following)"
 
     def GET(self, username, key=""):
-        mb = MyBooksTemplate(username, 'following')
+        i = web.input(page=1)
+        page_size = 25
+        offset = (max(0, int(i.page) - 1)) * page_size
```

Comment:
I don't know how necessary this is, but it makes invalid page numbers like ?page=0, ?page=-1, etc just defaults to the first page.